README.md

```diff
@@ -6,17 +6,34 @@
 
 [![GitHub](https://img.shields.io/github/license/pjhartzell/raster-footprint?style=for-the-badge)](https://github.com/pjhartzell/raster-footprint/blob/main/LICENSE)
 
-Create GeoJSON geometries that bound valid (not "nodata" pixels) raster data. Depends on [rasterio](https://rasterio.readthedocs.io/en/stable/) and [shapely](https://shapely.readthedocs.io/en/stable/manual.html).
+Create GeoJSON geometries that bound valid raster data. Depends on [rasterio](https://rasterio.readthedocs.io/en/stable/) and [shapely](https://shapely.readthedocs.io/en/stable/manual.html).
 
 ## Usage
 
 ```shell
 pip install raster-footprint
 ```
 
-Import and call one of the footprint functions with some options:
+Create or manipulate GeoJSON using the `rf` command:
```

Comment:
Bold choice to go with `rf`. _Personally_ I don't mind longer names so lean towards `raster-footprint` but this is your party 😄.

Reply:
+1 on bold choice. 🙂  I think you're right, though. `rf` is a bit abstract. It makes sense to use the same name as the package.